projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -46,6 +46,21 @@ public static function register() {
 		}
 	}
 
+	/**
+	 * Registers inline scripts which populate `window.JetpackWoAData` in Javascript.
+	 *   window.JetpackWoAData.isAtomicSite (Boolean)  - Is the site atomic?
+	 *   window.JetpackWoAData.isPrivateSite (Boolean) - Is the site private?
+	 *
+	 * @return void
+	 **/
+	public static function inject_javascript_options() {
```

Comment:
Instead of inserting them here, I think we could add them to `jetpack-blocks-editor` right away, here:
https://github.com/Automattic/jetpack/blob/12c1ec934857a0c0fa2afe2ddecbddb7cdbf2e4e/projects/plugins/jetpack/class.jetpack-gutenberg.php#L726-L739